cristin-project/src/test/java/no/unit/nva/cristin/projects/create/CreateCristinProjectHandlerTest.java

```diff
@@ -228,26 +229,47 @@ void shouldReturnCreatedWhenHealthProjectDataHasPartialData() throws Exception {
 
     @Test
     void shouldHaveProjectWithHealthDataAddedAndSentToUpstream() throws Exception {
-        var apiClient = new CreateCristinProjectApiClient(mockHttpClient);
-        apiClient = spy(apiClient);
-        handler = new CreateCristinProjectHandler(apiClient, environment);
+        var apiClient = createApiClientAndConnectToHandler();
 
         var nvaProject = randomNvaProject();
         nvaProject.setId(null);
         var healthProjectData = new HealthProjectData(DRUGSTUDY, null, CLINICAL_TRIAL_PHASE_VALUE);
         nvaProject.setHealthProjectData(healthProjectData);
 
+        var capturedCristinProject = captureCristinProjectFromApiClient(apiClient, nvaProject);
+
+        assertThat(capturedCristinProject.getHealthProjectType(),
+                   equalTo(nvaProject.getHealthProjectData().getType()));
+        assertThat(capturedCristinProject.getClinicalTrialPhase(),
+                   equalTo(nvaProject.getHealthProjectData().getClinicalTrialPhase()));
+    }
+
+    @Test
+    void shouldHaveProjectWithExemptFromPublicDisclosureAddedAndSentToUpstream() throws Exception {
```

Comment:
What happens when an unauthenticated user requests this?

Reply:
And creation of projects in proxy requires having access rights